third-party/genuine-fantasy-press/forgotten-secrets/items.xml

```diff
@@ -0,0 +1,1324 @@
+			<set name="rarity">Very Rare</set>
+			<set name="slot">onehand</set>
+		</setters>
+	</element>
+
+	<element name="Kaba Gaida of Nevihta" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_CIMBALOM_OF_DAZD">
+		<description>
+			<p>This bagpipe-like instrument was made using the bones and organs of a blue dragon. When played by someone with proficiency in a woodwind instrument, it enhances their spellcasting. While playing the kaba gaida, spells you cast that deal lightning or thunder damage are cast as though using a spell slot of one level higher. Additionally, you can cast gust of wind at will without expending a spell slot. Charisma is your spellcasting ability for this spell.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="0">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Very Rare</set>
+			<set name="attunement">true</set>
+			<set name="slot">onehand</set>
+		</setters>
+	</element>
+
+	<element name="Ancient Heart" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_ANCIENT_HEART">
+		<description>
+			<p>This is the heart of an elder being, pulsing with eldritch power and quivering with unnatural intent. You can consume it as an action. Roll a d6. You can increase one of your ability scores by a number of points equal to half the result (rounded down) and increase the maximum by the same amount. If you roll a 1, you die, your corpse is transformed into an unspeakable creature of CR 20 or greater, and you cannot be resurrected.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="5">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+		</setters>
+	</element>
+
+	<element name="Ancient Heart (Strength +1)" type="Magic Item" source="The Compendium of Forgotten Secrets" id="ID_GFP_COFSA_MAGIC_ITEM_WONDROUS_ITEM_ANCIENT_HEART_STRENGTH_ONE">
+		<description>
+			<p>This is the heart of an elder being, pulsing with eldritch power and quivering with unnatural intent. You can consume it as an action. Roll a d6. You can increase one of your ability scores by a number of points equal to half the result (rounded down) and increase the maximum by the same amount. If you roll a 1, you die, your corpse is transformed into an unspeakable creature of CR 20 or greater, and you cannot be resurrected.</p>
+		</description>
+		<setters>
+			<set name="category">Wondrous Items</set>
+			<set name="cost" currency="gp">0</set>
+			<set name="weight" lb="5">0 lb.</set>
+			<set name="type">Wondrous Item</set>
+			<set name="rarity">Legendary</set>
+			<!-- consumable items that apply rules not implemented, equip this item as alternative to activate it -->
```

Comment:
add `slot` setter to these items that have rules
without a `slot` or ` attunement` setter it will not be possible to equip these items